### ISSUE TITLE
Text within structured beyond first block are not translated

### ISSUE THREAD

**Issue by PaulBouisset** [voorhoede/datocms-plugin-translate-fields#13]
Hi, like asked I create a separate issue from [#12](https://github.com/voorhoede/datocms-plugin-translate-fields/issues/12)

If you translate a structured text the translation only work on the text before the first block in the structured text and not all the text content. 
If you don't have any blocks in your structured text that works but in case there is one block everything after the block is not translated.

**Comment by Siilwyn** [comment]
Thank you!

**Comment by PaulBouisset** [comment]
After contacting the support from dato it seems that using the dato function `buildBlockRecord ` with an no id should work to duplicate the blocks

**Comment by PaulBouisset** [comment]
[Siilwyn](https://github.com/Siilwyn) How can I push a fix branch for this bug? I implemented a fix in one line in the `getStructuredTextTranslation` function 
```
  const filteredArray = value.map(item => item.type !== 'block' ? item : {type:'block'})
```

**Comment by Siilwyn** [comment]
Awesome! @PaulBouisset you can create a PR by making a fork like [described in the GH Docs](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
If this feels too difficult for you we can also apply your suggested fix.

**Comment by PaulBouisset** [comment]
@Siilwyn I just did the pull request, sorry it was the first time I contributed to a public repo :) 
Hope the fix helps other people because for me it was blocking on a project. Thanks again for the amazing plugin 🙏

**Comment by Siilwyn** [comment]
Fixed by https://github.com/voorhoede/datocms-plugin-translate-fields/pull/14, published in v1.2.1

❤️ @PaulBouisset
